girder/cli/build.py

```diff
@@ -58,7 +58,7 @@ def main(dev, watch, watch_plugin, reinstall):
         if os.path.exists(npmLockFile):
             os.unlink(npmLockFile)
 
-        check_call(['npm', 'install'], cwd=staging)
+        check_call(['npm', 'install'], cwd=staging, shell=hasattr(sys, 'getwindowsversion'))
```

Comment:
@giesberge could you help me understand this change? What problem is being solved here for Windows that is fixed by running in a shell?

Reply:
@zachmullen Without using the shell, python subprocess reports that it can't find the `npm` command, even though on Windows, it is available via `where`.  With the shell it is resolved properly.   Oddly, `subprocess.check_call("where npm")` reports where it is, but `subprocess.check_call("npm")` can't find it, though `subprocess.check_call("npm", shell=True)` can.

Reply:
It turns out this is because nodejs uses a `.com` file for `npm.com`, and `.com` files can't be executed directly, only through a shell.  If it was `npm.exe`, it would work as expected.

Reply:
This feels like an npm (or windows?) bug, however we did used to support this via the now removed `--npm` option.

I think the best thing would be to add back `--npm`, I can't remember why it was removed.